src/main/python/deep_qa/models/reading_comprehension/attention_sum_reader.py

```diff
@@ -143,3 +143,13 @@ def _set_max_lengths(self, max_lengths: Dict[str, int]):
     def _set_max_lengths_from_model(self):
         self.max_sentence_length = self.model.get_input_shape_at(0)[1]
         # TODO(matt): implement this correctly
+
+    @classmethod
+    def _get_custom_objects(cls):
+        custom_objects = super(AttentionSumReader, cls)._get_custom_objects()
+        custom_objects["Attention"] = Attention
+        custom_objects["L1Normalize"] = L1Normalize
+        custom_objects["OptionAttentionSum"] = OptionAttentionSum
+        from ...layers.time_distributed_embedding import TimeDistributedEmbedding
```

Comment:
good question, i'm not too sure either --- the model import complains that it needs it, though.